snakemake/Snakefile

```diff
@@ -140,9 +140,9 @@ rule collapse_train_test_data:
         # Read input files, sort them, remove duplicates and write them out
         # Note: We use python instead of sort | uniq > to avoid locale specific issues
         # on the command line, and get a deterministic sort
-        'python -c "import sys; lines = sorted(set(line for file_path in sorted(sys.argv[1:]) for line in open(file_path, 'r'))); sys.stdout.writelines(lines)" {input.train_file} > {output.train_file} && '
-        'python -c "import sys; lines = sorted(set(line for file_path in sorted(sys.argv[1:]) for line in open(file_path, 'r'))); sys.stdout.writelines(lines)" {input.test_file} > {output.test_file} &&'
-        # For carbon_file, get header from first file and ignore it in the rest
+        # Get header from first file and ignore it in the rest
```

Comment:
All test/train files now have a header (since they're not just smiles, but smiles+inchikeys), so this change is needed.